lib/caps/DG2/vaon12

```diff
@@ -0,0 +1,94 @@
+###
+### Copyright (C) 2023 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+###
+### kate: syntax python;
+###
+
+# https://devblogs.microsoft.com/directx/video-acceleration-api-va-api-now-available-on-windows
+# https://github.com/intel/media-driver/blob/master/docs/media_features.md
```

Comment:
This link only applies to the iHD driver.  Does vaon12 driver eventually route into the iHD driver on Windows?  Maybe you can share a software stack architecture diagram?